File: DayTwo/CeasarCipher.java
public class CeasarCipher{
    public static void main (String[] args){
        String text = "HELLO";
        int shift = 3;
        String encrypted = "";
    
        for(int index =0; index < text.length(); index++){
            char ch = text.charAt(index);
            if(ch >= 'A' && ch <= 'Z'){
                ch = (char)(ch +shift);

            if(ch >'Z'){
                ch = (char)(ch-26);
              }
            
            }
                encrypted += ch;
        } 
        System.out.println("the encrypted Text is : " +encrypted);
    }
}

File: DayTwo/ceasar_cipher.py
def getEncryption(word, shift):
    encrypted = ""
    for index in range(len(word)):
        ch = word[index] 
        
        if 'A' <= ch <= 'Z':
            ch= chr(ord(ch) + shift)

            if ch > 'Z':
                ch = chr(ord(ch)-26)


        encrypted += ch
return encrypted

def getDecryption(word, shift):
    dcrypted= ""

    for index in range(len(word)):
    ch = word[index]

    if'A' <= ch <= 'Z':
        ch = chr(ord(ch)-shift)
        
        if ch > 'Z':
            ch = chr(ord(ch)+ 26)
    dcrypted += ch
return dcrypted
        
        

